BraintreeCore/src/main/java/com/braintreepayments/api/PaymentMethodClient.java

```diff
@@ -70,7 +70,7 @@ public PaymentMethodClient(BraintreeClient braintreeClient) {
      *        payment methods will be ordered by most recently added.
      * @param callback {@link GetPaymentMethodNoncesCallback}
      */
-    public void getPaymentMethodNonces(boolean defaultFirst, final GetPaymentMethodNoncesCallback callback) {
+    void getPaymentMethodNonces(boolean defaultFirst, final GetPaymentMethodNoncesCallback callback) {
```

Comment:
This seems like a method that would be useful to merchants

Reply:
Ok I wasn't sure on this - I couldn't find any documentation in our dev docs on any of the `PaymentMethodClient` methods, and our only usage of them was in Drop-in.  If merchants do need these methods I can put them back to public.

Reply:
We have a [similar public method on iOS](https://github.com/braintree/braintree_ios/blob/8b56f308f69ca4500a3c699fd1d2f55f2d6891f3/Sources/BraintreeCore/Public/BraintreeCore/BTAPIClient.h#L85-L94) `fetchPaymentMethodNonces`. 

May as well keep things consistent and have it be public on both (or hidden on both). I can see this being useful to merchants.